src/procedures/the-cat-api/the-cat-api.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * The Cats Service provides access to the Cat API data from https://api.thecatapi.com/v1/.
+ *
+ * @service
+ * @category Society
+ */
+
+
+const { TheCatApiKey } = require('../utils/api-key');
+const {catTypes} = require('./types');
+const ApiConsumer = require('../utils/api-consumer');
+catTypes();
+
+// Cats API Url
+const catApiUrl = 'https://api.thecatapi.com/v1/images/search';
+
+ const TheCatApi = new ApiConsumer('TheCatApi', catApiUrl, {cache: {ttl: 1}});
+ ApiConsumer.setRequiredApiKey(TheCatApi, TheCatApiKey)
+
+/**
+ * Get random cat image.
+ * @param {BreedsOfCats=} catBreeds The list of all possible Cat Breeds filterable.
+ * @returns {Image} the requested image
+ */
+TheCatApi.getRandomCatImage = async function(catBreeds = ''){
```

Comment:
Isn't `catBreeds` a single breed? I was a little thrown off here initially wondering if it was a comma-separated list or something.